schema/Figure.schema.yaml

```diff
@@ -0,0 +1,28 @@
+title: Figure
+'@id': stencila:Figure
+extends: CreativeWork
+role: primary
+status: experimental
+category: prose
+description: Encapsulates one or more images, videos, tables, etc, and provides captions and labels for them.
+properties:
+  label:
```

Comment:
I'm not convinced that a `label` property is necessary, given that we have `caption` and/or `content`.

Reply:
I was thinking of a case where if it’s undefined then we generate it. Otherwise we display what the user has put in. Without a field for this we’d lose it going back and forth to JATS?

Reply:
That's sounds like a good approach to me.